config/bind9/README.md

```diff
@@ -0,0 +1,245 @@
+
+;; ADDITIONAL SECTION:
+ns.example.com.         30      IN      A       127.0.0.1
+```
+
+At this point you have a working Bind9 instance running that we can query.
+
+## Configure kuadrant CoreDNS zone (k.example.com) delegation
+
+Check for CoreDNS instances currently running on the cluster:
+
+```shell
+kubectl get service -A -l app.kubernetes.io/name=coredns,app.kubernetes.io/component!=metrics
+```
+Example output:
+```
+NAMESPACE          NAME               TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                     AGE
+kuadrant-coredns   kuadrant-coredns   LoadBalancer   10.96.242.254   172.18.0.16   53:31680/UDP,53:31680/TCP   31m
+```
+
+Each of the CoreDNS instances should be watching for records in the "k.example.com" zone. You can verify this by checking the configuration or logs:
+```
+kuadrant-coredns kuadrant-coredns-5fd84d57b5-dqbcf coredns [INFO] plugin/kuadrant: Starting informer 0 for zone k.example.com.
+```
+
+Get a list of all the CoreDNS services external ips:
+```shell
+kubectl get service -A -l app.kubernetes.io/name=coredns,app.kubernetes.io/component!=metrics -o json | jq -r '[.items[] | (.status.loadBalancer.ingress[].ip)] | join(",")'
+```
+Example output:
+```
+172.18.0.16
+```
+**Note:** In the case of multiple deployments this would be a comma seperated list of IPs.
+
+For each IP create a nsupdate command and issue it against the edge server to add the zone records.
+
+To generate a nsupdate file for the first CoreDNS instance found:
+```shell
+EDGE_NS=`kubectl get service/kuadrant-bind9 -n kuadrant-bind9 -o json | jq -r '.status.loadBalancer.ingress[].ip'`
+CORE_NS=`kubectl get service -A -l app.kubernetes.io/name=coredns,app.kubernetes.io/component!=metrics -o json | jq -r '[.items[] | (.status.loadBalancer.ingress[].ip)][0]'`
+cat <<EOF >>nsupdate-k-example-com-1
+server ${EDGE_NS}
+debug yes
+zone example.com.
+update add k.example.com 300 IN NS ns1.k.example.com
+update add ns1.k.example.com 300 IN A ${CORE_NS}
+send
+EOF
+```
+
+Check the resulting nsupdate file:
+```shell
+cat nsupdate-k-example-com-1
+```
+Example output:
+```
+server 172.18.0.17
+debug yes
+zone example.com.
+update add k.example.com 300 IN NS ns1.k.example.com
+update add ns1.k.example.com 300 IN A 172.18.0.16
+send
+```
+
+Apply the update:
+```shell
+nsupdate -k ddns.key -v nsupdate-k-example-com-1
+```
+Example output:
+```
+Sending update to 172.18.0.17#53
+Outgoing update query:
+;; ->>HEADER<<- opcode: UPDATE, status: NOERROR, id:  35292
+;; flags:; ZONE: 1, PREREQ: 0, UPDATE: 2, ADDITIONAL: 1
+;; ZONE SECTION:
+;example.com.                   IN      SOA
+
+;; UPDATE SECTION:
+k.example.com.          300     IN      NS      ns1.k.example.com.
+ns1.k.example.com.      300     IN      A       172.18.0.16
+
+;; TSIG PSEUDOSECTION:
+example.com-key.        0       ANY     TSIG    hmac-sha256. 1746017622 300 32 KcrBVlBh3qMii9PnVdyo/yvL5nyNyzRd/0UMh63NXCU= 35292 NOERROR 0 
```

Comment:
This could be scripted, but i think time would be better spent adding the nsupdate provider https://github.com/Kuadrant/dns-operator/issues/455